test/unit/test_util.c

```diff
@@ -113,6 +113,41 @@ bool test_autonames(void) {
 	mu_end;
 }
 
+bool test_file_slurp(void) {
+
+#ifdef __WINDOWS__
+#define S_IRWXU _S_IREAD | _S_IWRITE
+#endif
+
+	const char* test_file =  "./empty_file";
+	size_t s;
+	const char* some_words = "some words";
+
+    int f = open (test_file, O_CREAT, S_IRWXU);
+    mu_assert_neq (f, -1, "cannot create empty file");
+    close (f);
+
+	char* content = r_file_slurp (test_file, &s);
+	mu_assert_eq (s, 0, "size should be zero");
+	mu_assert_eq (strlen(content), 0, "returned buffer should be empty");
+	free (content);
+
+	f = open (test_file, O_WRONLY, S_IRWXU);
+    mu_assert_neq (f, -1, "cannot reopen empty file");
```

Comment:
Pay attention to style please

Reply:
Oups, I keep forgetting due to my vim setting :/